package.json

```diff
@@ -24,7 +24,8 @@
   },
   "main": "./index",
   "peerDependencies": {
-    "mocha": "^2.0.0"
+    "mocha": "^2.0.0",
+    "karma-mocha": "^1.1.1"
```

Comment:
karma-mocha is not peer dependency. Library can be used without `karma` and without `karma-mocha`. Sure, this is de-facto standard but `karma-mocha` is not a dependency for this lib